### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by maximpristupa**

### CODE REVIEW

File: server/rw.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    });
+    let numbers = objects.map(function(num) {
+        return num.name.match(/\d/)[0];
+    });
+    let lastLetter = letters.sort().pop();
+    let lastNumber = numbers.sort().pop();
+    let file = [];
+    for (var i = 0; i < lastNumber; i++) {
+        let row = [];
+        for (var y = 'A'.charCodeAt(0); y < lastLetter.charCodeAt(0) + 1; y++) {
```
Comment (YanaYaroshevich) [inline server/rw.js]: do it before loop and assign to variables
'A'.charCodeAt(0)
lastLetter.charCodeAt(0)

File: server/rw.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            let found = objects.find(function(element) {
```
Comment (YanaYaroshevich) [inline server/rw.js]: lambda functions pls

File: server/rw.js (hunk 1 above)
Comment (maximpristupa) [reply server/rw.js]: done

File: server/rw.js (hunk 2 above)
Comment (maximpristupa) [reply server/rw.js]: done

File: server/rw.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+// Data
+
+const fs = require('fs');
+const charCodeA = 'A'.charCodeAt(0);
+// Write
+
+function objectsToCsv(objects) {
+    let letters = objects.map(function(num) {
```
Comment (YanaYaroshevich) [inline server/rw.js]: (num) => num.name.match(/[A-Z]/)[0]

Comment (YanaYaroshevich) [reply server/rw.js]: const

File: server/sorting.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+const calc = require('./evaluate');
+let calculatedСells = [];
+
+function csvToObjects(csv) {
+    let objects = [];
+    calculatedСells = [];
+    let objectType = typeof csv;
+    let lines;
+    if (objectType == 'string') {
+        lines = csv.split("\n");
```
Comment (YanaYaroshevich) [inline server/sorting.js]: single quotes

File: server/sorting.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        lines = csv;
+    }
+    lines.forEach((line, lineIndex) => {
+        let cells = line.split(",");
+        cells.forEach((cell, cellIndex) => {
+            if (!cell) {
+                return;
+            }
+            let links = cell.match(/[A-Z]\d/g);
+            if (cell != `${+cell}` && cell[0] != '=') {
```
Comment (YanaYaroshevich) [inline server/sorting.js]: !==

**Pull request merged**